templates/addons/versionControl/github/.github/workflows/lint.yml

```diff
@@ -25,16 +21,11 @@ jobs:
         with:
           ref: ${{ github.head_ref }}
 
-      - name: Install Terraform
-        uses: hashicorp/setup-terraform@v2
-        with:
-          terraform_version: ${{ env.TERRAFORM_VERSION }}
+      - name: Install dependencies in .tool-versions
+        uses: asdf-vm/actions/install@v2
 
       - name: Run Terraform format
         run: terraform fmt -recursive -check
 
-      - name: Run tfsec linter
-        id: tfsec
-        uses: aquasecurity/tfsec-action@v1.0.3
-        with:
-          version: ${{ env.TFSEC_VERSION }}
+      - name: Run trivy linter
```

Comment:
```suggestion
      - name: Run trivy scanner
```

Reply:
Update them all in [a518570](https://github.com/nimblehq/infrastructure-templates/pull/262/commits/a51857096f78c1142f953c3f54351c668daf93a0)